src/layer-events-hoc.tsx

```diff
@@ -132,57 +137,118 @@ function layerMouseTouchEvents(
       const { map } = this.context;
 
       evt.features.forEach((feature: Feature) => {
+        // Set hover
         const { id } = feature.properties;
         this.hover = [id];
-        if (this.areFeaturesDraggable(children)) {
-          map.dragPan.disable();
-          this.isDragging = true;
 
-          map.on('touchmove', this.onDragMove);
-          map.once('touchend', this.onDragUp.bind(this, 'touchmove'));
+        if (!this.areFeaturesDraggable(children)) {
```

Comment:
This was an issue, instead of checking all the children feature `!this.areFeaturesDraggable(children)`, we should check for the current touched feature, something like: `!this.isFeatureDraggable(feature)`. Can we change this in the meantime?